contrib/log/aliyun/aliyun.go

```diff
@@ -157,10 +158,8 @@ func toString(v interface{}) string {
 		key = strconv.FormatInt(v, 10)
 	case uint64:
 		key = strconv.FormatUint(v, 10)
-	case string:
-		key = v
```

Comment:
Why did you delete this?

Reply:
type `string` implements fmt.Stringer interface.

Reply:
I tested, type `string` does not implement the `fmt.Stringer` interface.